### PR TITLE
Wait timeout fix

### PR DESCRIPTION
**Pull request by rmweir**

### CODE REVIEW

File: apis/project.cattle.io/v3/app_types.go [hunk 1]
```diff
@@ -17,16 +17,19 @@ type App struct {
...
-	MultiClusterAppName string            `json:"multiClusterAppName,omitempty" norman:"type=reference[/v3/schemas/multiclusterapp]"`
-	ValuesYaml          string            `json:"valuesYaml,omitempty"`
+	ProjectName     string            `json:"projectName,omitempty" norman:"type=reference[/v3/schemas/project]"`
+	Description     string            `json:"description,omitempty"`
+	TargetNamespace string            `json:"targetNamespace,omitempty"`
+	ExternalID      string            `json:"externalId,omitempty"`
+	Files           map[string]string `json:"files,omitempty"`
+	Answers         map[string]string `json:"answers,omitempty"`
+	Wait            bool              `json:"wait,omitempty"`
+	// Timeout is a pointer to differentiate between a set 0 and the default
```
Comment (cbron) [inline apis/project.cattle.io/v3/app_types.go]: Still necessary ?

Comment (cbron) [reply apis/project.cattle.io/v3/app_types.go]: fixed but can't mark this as resolved for some reason.

**Comment by luthermonson** [comment]
lgtm

**Comment by dramich** [comment]
lgtm

**Pull request merged**